EnergyStar/settings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "exemptions": [
+    //Not ourselves
```

Comment:
I would keep comment in the configuration (and I already saw you set the JSON parser to omit comments, nice) so users can understand the rationale of default configuration.